frontend/src/views/shared/Widgets/WidgetCategoricalChart.jsx

```diff
@@ -0,0 +1,194 @@
+            </h3>
+          </div>
+        ) : (
+          <>
+            <h4>
+              <T _str="PIXELS COUNTS BY CATEGORY" />
+            </h4>
+            <div className="categories">
+              <T _str="CATEGORIES" />
+            </div>
+            <ResponsiveContainer width={width} height={height}>
+              <BarChart
+                layout="vertical"
+                data={mergedBarData}
+                margin={{ bottom: CHART_MARGIN_BOTTOM }}
+              >
+                <XAxis
+                  dataKey="count"
+                  type="number"
+                  interval={0}
+                  label={() => (
+                    <text
+                      x={-5}
+                      y={height + 10 - CHART_MARGIN_BOTTOM}
+                      className="categorical-x-axis-label"
+                    >
+                      <T _str="(No. of pixels)" />
+                    </text>
+                  )}
+                />
+                <YAxis
+                  dataKey="name"
+                  interval={0}
+                  axisLine={false}
+                  type="category"
+                  mirror
+                  tick={false}
+                  tickLine={false}
+                />
+
+                <Tooltip content={<CustomTooltip unit={unit} minimumFractionDigits={0} />} />
+                <Bar dataKey="count" unit={unit} shape={<CustomBar chartWidth={width} />} />
+              </BarChart>
+            </ResponsiveContainer>
+
+            {data.stats && (
+              <ul className="m-widget__stats">
+                <li className="stats-item">
+                  <span className="stats-item__label">
+                    <T _str="Max:" _comment="Max number on the widget bar chart" />{' '}
+                  </span>
+                  {formatNumber({ value: data.stats.max })}&nbsp;
+                  {unit}
+                </li>
+                <li className="stats-item">
+                  <span className="stats-item__label">
+                    <T _str="Min:" _comment="Min number on the widget bar chart" />{' '}
+                  </span>
+                  {formatNumber({ value: data.stats.min })}&nbsp;
+                  {unit}
+                </li>
+                <li className="stats-item">
+                  <span className="stats-item__label">
+                    <T _str="Std. deviation:" _comment="Std. deviation: on the widget bar chart" />{' '}
+                  </span>
+                  {formatNumber({ value: data.stats.std })}&nbsp;
+                  {unit}
+                </li>
+                <li className="stats-item">
+                  <span className="stats-item__label">
+                    <T _str="Sum:" _comment="Sum: on the widget bar chart" />{' '}
+                  </span>
+                  {formatNumber({ value: data.stats.sum })}&nbsp;
+                  {unit}
+                </li>
+              </ul>
+            )}
```

Comment:
I'm not sure these stats make sense for this chart and there are not part of the design so I think we could remove them.